src/Uno.UI/UI/Xaml/FrameworkElement.cs

```diff
@@ -122,6 +122,10 @@ Windows.UI.Xaml.ResourceDictionary Resources
 		/// <returns>The size that this object determines it needs during layout, based on its calculations of the allocated sizes for child objects or based on other considerations such as a fixed container size.</returns>
 		protected virtual Size MeasureOverride(Size availableSize)
 		{
+#if !__WASM__
+			LastAvailableSize = availableSize;
```

Comment:
I don't  remember where ... but I'm pretty sure that we already have a property equivalent to this `LastAvailableSize` ... maybe @davidjohnoliver knows where ?

Reply:
There's one for Wasm, but there's none for iOS/Android.